crates/diesel_models/src/payment_intent.rs

```diff
@@ -197,7 +197,7 @@ common_utils::impl_to_sql_from_sql_json!(TaxDetails);
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
 pub struct PaymentMethodTypeTax {
     pub order_tax_amount: MinorUnit,
-    pub pmt: PaymentMethodType,
+    pub payment_method_type: PaymentMethodType,
```

Comment:
if we are storing this in the database, then previous values which are already stored in the database might not be readable, can you confirm that once? Create a payment using the older version and try to retrieve it in the newer version and vice versa